apps/aecore/src/aec_tx_pool.erl

```diff
@@ -168,13 +168,17 @@ push(Tx, Event) ->
 
 -spec push(aetx_sign:signed_tx(), event(), push_timeout()) -> ok | {error, atom()}.
 push(Tx, Event = tx_received, Timeout) ->
+    lager:debug("[domat] tx_received", []),
```

Comment:
'domat'?

Reply:
oh no, my secret logging message is revealed. Again...